tools/SourceKit/cmake/modules/AddSwiftSourceKit.cmake

```diff
@@ -161,8 +161,11 @@ function(add_sourcekit_swift_runtime_link_flags target path HAS_SWIFT_MODULES)
       # Add rpath to the host Swift libraries.
       file(RELATIVE_PATH relative_hostlib_path "${path}" "${SWIFTLIB_DIR}/host")
       list(APPEND RPATH_LIST "$ORIGIN/${relative_hostlib_path}")
+    else()
+      target_link_directories(${target} PRIVATE
+        ${SWIFT_PATH_TO_SWIFT_SDK}/usr/lib/swift/Windows/x86_64)
     endif()
-
+  else()
```

Comment:
This block was only added for SWIFT_SWIFT_PARSER. BOOTSTRAPPING= is handled above, it was copy pasted so the comments don't really make much sense. TLDR: no `else` here please.

Reply:
I needed the else case otherwise it didn't build IIRC

Reply:
Where? The block below only runs on Darwin and needs to keep running without the `else`. Is the `set(RPATH_LIST ${RPATH_LIST} PARENT_SCOPE)` the issue?

Reply:
```
FAILED: bin/sourcekitdInProc.dll lib/sourcekitdInProc.lib
cmd.exe /C "cd . && "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -E vs_link_dll --intdir=tools\swift\tools\SourceKit\tools\sourcekitd\bin\InProc\CMakeFiles\sourcekitdInProc.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~2\2022\BUILDT~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\link.exe /nologo @CMakeFiles\sourcekitdInProc.rsp  /out:bin\sourcekitdInProc.dll /implib:lib\sourcekitdInProc.lib /pdb:bin\sourcekitdInProc.pdb /dll /version:0.0 /machine:x64 /INCREMENTAL:NO  /DEF:"S:/b/1/tools/swift/tools/SourceKit/tools/sourcekitd/bin/InProc/sourcekitdInProc.def"  --target=x86_64-unknown-windows-msvc  && cd ."
LINK: command "C:\PROGRA~2\MICROS~2\2022\BUILDT~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\link.exe /nologo @CMakeFiles\sourcekitdInProc.rsp /out:bin\sourcekitdInProc.dll /implib:lib\sourcekitdInProc.lib /pdb:bin\sourcekitdInProc.pdb /dll /version:0.0 /machine:x64 /INCREMENTAL:NO /DEF:S:/b/1/tools/swift/tools/SourceKit/tools/sourcekitd/bin/InProc/sourcekitdInProc.def --target=x86_64-unknown-windows-msvc /MANIFEST /MANIFESTFILE:bin\sourcekitdInProc.dll.manifest" failed (exit code 1104) with the following output:
LINK : warning LNK4044: unrecognized option '/-target=x86_64-unknown-windows-msvc'; ignored
LINK : fatal error LNK1104: cannot open file 'swift_Concurrency.lib'
```